### ISSUE TITLE
[BUG]CSS  may be updated in 0.15.4

### ISSUE THREAD

**Issue by Quorafind** [AlexBieg/obsidian-better-command-palette#67]
![image](https://user-images.githubusercontent.com/13215013/177448179-3ebbe2b2-c5e1-4510-9dbc-aac0eefbde18.png)

Hi, in 0.15.4 version of Obsidian. Suggest items’ css changed.

- For plugin and theme developers: we've made changes to the Suggest component's DOM structure. If your plugin is attaching itself to core Suggest components or mimicking the Suggest component structure, it will likely be impacted.
    - The new DOM structure is:

```  
    - .suggestion-item
        - .suggestion-content
            - .suggestion-title
            - .suggestion-note (optional)
        - .suggestion-aux
            - .suggestion-hotkey (optional)
            - .suggestion-flair (optional)
```

**Comment by brimwats** [comment]
related to #68

**Comment by AlexBieg** [comment]
This should be fixed in v14.0.0
